packages/xstate-wallet/src/workflows/conclude-channel.ts

```diff
@@ -83,6 +83,7 @@ const virtualDefunding = {
   onDone: 'success'
 };
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const withdraw = (store: Store) => async (ctx: Init) => {
```

Comment:
We can't remove the param?

Reply:
In this particular case I'm not sure why `withdraw` is a "no-op". @andrewgordstewart @lalexgap ?